deps/v8/tools/gen-postmortem-metadata.py

```diff
@@ -382,7 +383,7 @@ def load_objects():
 
 
 def load_objects_from_file(objfilename, checktypes):
-        objfile = open(objfilename, 'r');
+        objfile = io.open(objfilename, 'r', encoding='utf-8')
```

Comment:
Would it be better to keep trailing `;` like rest of the file?

Reply:
Has addressed in https://chromium-review.googlesource.com/c/v8/v8/+/1895560.

Marking outdated...

Reply:
For the record, semicolons are not Pythonic.